code/style.css

```diff
@@ -1,4 +1,379 @@
+
+p {
+    font-family: 'Montserrat', sans-serif;
+    color: #214273;  
+    line-height: 1.5;  
+}
+
+/* HEADER */
+
+.header-parent {
+    width: 100%;
+    height: 100px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    padding-top: 10px;
+    box-sizing: border-box;
+}
+
+.logo-title{
+    box-sizing: border-box;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding-left: 10px;
+}
+
+.logo-title h1 {
+    margin-left: 10px;
+    font-size: 28px;
+}
+
+.hamburger-menu {
+    width: 30px;
+    height: 30px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    margin: 24px;
+    box-sizing: border-box;
+}
+    
+.hamburger-menu div {
+    width: 30px;
+    height: 3px;
+    background: #214273;
+    border-radius: 2px;
+}
+
+.header-parent nav {
+    display: none;
+}
+
+
+/* MAIN ARTICLE */
+
+.main-article-parent {
+    width: 100%;
+    background:white;
+    padding-bottom: 10px;
+}
+
+.main-article-child {  
+    width: 100%;
+    padding: 50px 15px;
+    background: rgb(255, 215, 222); 
+    box-sizing: border-box;  
+    margin: auto; 
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+}
+
+.main-article-child img {
+    width: 100%;
+    border: 10px solid white;
+    box-sizing: border-box;
+}
+
+.main-article-header {
+    border-bottom: solid 1px #214273;
+    padding: 0px 0px 10px 0px;
+    box-sizing: border-box;
+    margin: 0;
+}
+
+.main-article-child div h2 a {
+    color: #214273;
+    text-decoration: none;
+}
+
+.main-article-child div h2 a:hover {
+    color: #fa5aaa;
+}
+
+.text-area {
+    background: white;
+    margin-top: 10px;
+    padding: 15px 15px;
+}
+
+ .text-area p {
+    margin: auto;
+    padding: 10px 0px;
+    box-sizing: border-box;
+}
+
+.read-more a{
+    background: #ff8cc5;
+    text-decoration: none;
+    border-radius: 2px;
+    font-family: 'Montserrat', sans-serif; 
+    font-size: 10px;
+    font-weight: 600;
+    color:  #214273;
+    margin: 0 0;
+    padding: 5px;
+    box-sizing: border-box;
+}
+
+.read-more a:hover {
+    background: #fda7d2;
+    color: white;
+}   
+
+/* DIVIDER */
+
+.opn-divider {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    margin-bottom: 10px;
+}
+
+.opn-divider h1 {
+    font-size: 23px;
+    padding-bottom: 5px;
+    margin-bottom: 10px;
+    border-bottom: solid 1px #214273;
+    box-sizing: border-box;
+}
+
+.opn-divider img {
+    width: 40px;
+}
+
+/* OTHER ARTICLES */
+
+.other-article-parent {
+    width: 100%;
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: center;
+    padding: 10px 0px;
+}
+
+.other-article-child {  
+    width: 90%;
+    padding: 14px;
+    box-sizing: border-box;  
+    margin: 20px auto;  
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
```

Comment:
This is some next level ish! To be able to layer your box shadows and making it look like they smoothly "pop" is a super nice UI feature!